### PR TITLE
Update for improved command templates

### PR DESCRIPTION
**Pull request by sciencewhiz**
Remove CommandBase reference and use frc namespace
Add copyright statement and change to #pragma once
Fix Trigger compilation by removing const qualifier not in Trigger

### CODE REVIEW

**Comment by sciencewhiz** [comment]
Companion to https://github.com/wpilibsuite/allwpilib/pull/906

File: edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/Trigger.h [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+#pragma once
 
 #include <Buttons/Trigger.h>
 
-class $classname : public Trigger {
+class $classname : public frc::Trigger {
 public:
 	$classname();
-	bool Get() const;
+	bool Get();
```
Comment (calcmogul) [inline edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/Trigger.h]: I would make this line `bool Get() override;`.

File: edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/TimedCommand.h [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+
+#pragma once
 
 #include "Commands/TimedCommand.h"
 
-class $classname : public TimedCommand {
+class $classname : public frc::TimedCommand {
 public:
 	$classname(double timeout);
 	void Initialize();
```
Comment (calcmogul) [inline edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/TimedCommand.h]: These should have `override` appended as well.

File: edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/TimedCommand.h [hunk 3]
```diff
@@ -1,15 +1,19 @@
-#ifndef $classname_H
-#define $classname_H
+/*----------------------------------------------------------------------------*/
```
Comment (calcmogul) [inline edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/TimedCommand.h:1]: Do we want template files to have the WPILib copyright? If not, we can remove them and add a regex to the `licenseUpdateExclude` block in .styleguide.

**Review by calcmogul — CHANGES_REQUESTED** [review]
We may want to include `override` in more places. Even though it's on basically every function, it's good practice to do so.

File: edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/TimedCommand.h (hunk 3 above)
Comment (sciencewhiz) [reply edu.wpi.first.wpilib.plugins.cpp/resources/templates/filetemplates/command-based/TimedCommand.h:1]: I believe they've always had the wpilib copyright

**Review by calcmogul — APPROVED** [review]

**Pull request merged**